change quartz back to broadwell

PeterCaldwell opened the pull request:
Recent mods switched components/scream/cmake/machine-files/quartz.cmake from being BDW architecture to SKL, causing "illegal instruction" errors when scream tests were run.

bartgol reviewed (approved):
Whoops! Copy+paste error. Sorry!

E3SM-Autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Blake</h3>
<ul>
  <li>Build Num: 158</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: AUTOMERGE;B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Bowman</h3>
<ul>
  <li>Build Num: 154</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: AUTOMERGE;B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 156</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: AUTOMERGE;B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: PeterCaldwell/quartz-llnl_cmake_fix</li>
    <li>SHA: 977fe2af14a2bfb6e8186043056251173135c89e</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: PeterCaldwell
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: 1 or more Jobs FAILED

Note: Testing will normally be attempted again in approx. 4 Hrs. If a change to the PR source branch occurs, the testing will be attempted again on next available autotester run.

<details>
<summary>
  <b>Pull Request Auto Testing has FAILED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Blake</h3>
<ul>
  <li>Build Num: 158</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: AUTOMERGE;B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Bowman</h3>
<ul>
  <li>Build Num: 154</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: AUTOMERGE;B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 156</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>AT: AUTOMERGE;B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>
</details>


<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Blake # 158</b> (click to expand)
</summary>
<pre>
<code>
Running as SYSTEM
Building remotely on blake in workspace /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake
The recommended git tool is: NONE
using credential cf1815d3-62c3-4157-a38d-0ad235093618
Wiping out workspace first.
Cloning the remote Git repository
Cloning repository https://github.com/E3SM-Project/scream.git
 > /home/projects/x86-64/git/2.9.4/bin/git init /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/158/scream # timeout=10
Fetching upstream changes from https://github.com/E3SM-Project/scream.git
 > /home/projects/x86-64/git/2.9.4/bin/git --version # timeout=10
 > git --version # 'git version 2.9.4'
using GIT_SSH to set credentials jgfouca github key
Setting http proxy: wwwproxy.sandia.gov:80
 > /home/projects/x86-64/git/2.9.4/bin/git fetch --tags --progress -- https://github.com/E3SM-Project/scream.git +refs/heads/*:refs/remotes/origin/* # timeout=30
 > /home/projects/x86-64/git/2.9.4/bin/git config remote.origin.url https://github.com/E3SM-Project/scream.git # timeout=10
 > /home/projects/x86-64/git/2.9.4/bin/git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > /home/projects/x86-64/git/2.9.4/bin/git rev-parse refs/remotes/origin/pr/612^{commit} # timeout=10
 > /home/projects/x86-64/git/2.9.4/bin/git rev-parse pr/612^{commit} # timeout=10
 > /home/projects/x86-64/git/2.9.4/bin/git rev-parse pr/612^{commit} # timeout=10
ERROR: Couldn't find any revision to build. Verify the repository and branch configuration for this job.
Sending e-mails to: lbertag@sandia.gov
Finished: FAILURE

</code>
</pre>
</details>

<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Bowman # 154</b> (click to expand)
</summary>
<pre>
<code>
Running as SYSTEM
Building remotely on bowman in workspace /home/e3sm-jenkins/bowman/workspace/SCREAM_PullRequest_Autotester_Bowman
The recommended git tool is: NONE
using credential cf1815d3-62c3-4157-a38d-0ad235093618
Wiping out workspace first.
Cloning the remote Git repository
Cloning repository https://github.com/E3SM-Project/scream.git
 > /home/projects/x86-64-knl/git/2.8.2/bin/git init /home/e3sm-jenkins/bowman/workspace/SCREAM_PullRequest_Autotester_Bowman/154/scream # timeout=10
Fetching upstream changes from https://github.com/E3SM-Project/scream.git
 > /home/projects/x86-64-knl/git/2.8.2/bin/git --version # timeout=10
 > git --version # 'git version 2.8.3'
using GIT_SSH to set credentials jgfouca github key
Setting http proxy: wwwproxy.sandia.gov:80
 > /home/projects/x86-64-knl/git/2.8.2/bin/git fetch --tags --progress -- https://github.com/E3SM-Project/scream.git +refs/heads/*:refs/remotes/origin/* # timeout=30
 > /home/projects/x86-64-knl/git/2.8.2/bin/git config remote.origin.url https://github.com/E3SM-Project/scream.git # timeout=10
 > /home/projects/x86-64-knl/git/2.8.2/bin/git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > /home/projects/x86-64-knl/git/2.8.2/bin/git rev-parse refs/remotes/origin/pr/612^{commit} # timeout=10
 > /home/projects/x86-64-knl/git/2.8.2/bin/git rev-parse pr/612^{commit} # timeout=10
 > /home/projects/x86-64-knl/git/2.8.2/bin/git rev-parse pr/612^{commit} # timeout=10
ERROR: Couldn't find any revision to build. Verify the repository and branch configuration for this job.
Sending e-mails to: lbertag@sandia.gov
Finished: FAILURE

</code>
</pre>
</details>

<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Weaver # 156</b> (click to expand)
</summary>
<pre>
<code>
Running as SYSTEM
Building remotely on weaver in workspace /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver
The recommended git tool is: NONE
using credential cf1815d3-62c3-4157-a38d-0ad235093618
Wiping out workspace first.
Cloning the remote Git repository
Cloning repository https://github.com/E3SM-Project/scream.git
 > /home/projects/ppc64le/git/2.10.1/bin/git init /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/156/scream # timeout=10
Fetching upstream changes from https://github.com/E3SM-Project/scream.git
 > /home/projects/ppc64le/git/2.10.1/bin/git --version # timeout=10
 > git --version # 'git version 2.10.1'
using GIT_SSH to set credentials jgfouca github key
Setting http proxy: wwwproxy.sandia.gov:80
 > /home/projects/ppc64le/git/2.10.1/bin/git fetch --tags --progress -- https://github.com/E3SM-Project/scream.git +refs/heads/*:refs/remotes/origin/* # timeout=30
 > /home/projects/ppc64le/git/2.10.1/bin/git config remote.origin.url https://github.com/E3SM-Project/scream.git # timeout=10
 > /home/projects/ppc64le/git/2.10.1/bin/git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > /home/projects/ppc64le/git/2.10.1/bin/git rev-parse refs/remotes/origin/pr/612^{commit} # timeout=10
 > /home/projects/ppc64le/git/2.10.1/bin/git rev-parse pr/612^{commit} # timeout=10
 > /home/projects/ppc64le/git/2.10.1/bin/git rev-parse pr/612^{commit} # timeout=10
ERROR: Couldn't find any revision to build. Verify the repository and branch configuration for this job.
Sending e-mails to: lbertag@sandia.gov
Finished: FAILURE

</code>
</pre>
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Blake</h3>
<ul>
  <li>Build Num: 165</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Bowman</h3>
<ul>
  <li>Build Num: 159</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 161</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: SCREAM (E3SM-Project/scream)</dt>
  <ul>
    <li>Branch: PeterCaldwell/quartz-llnl_cmake_fix</li>
    <li>SHA: 977fe2af14a2bfb6e8186043056251173135c89e</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: PeterCaldwell
</details>

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Blake</h3>
<ul>
  <li>Build Num: 165</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Bowman</h3>
<ul>
  <li>Build Num: 159</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 161</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>B4B;CI: Integrate Without Testing;Machine File;bugfix</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>612</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>PeterCaldwell/quartz-llnl_cmake_fix</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>977fe2af14a2bfb6e8186043056251173135c89e</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>6d003f397a540d2bfbf5dec413ef39af523e3c6c</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>
</details>

E3SM-Autotester commented:
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ bartgol ]!

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - AutoMerge IS ENABLED, but the Label AT: AUTOMERGE is not set.  Either set Label AT: AUTOMERGE or manually merge the PR...

AaronDonahue commented:
should we add "AutoMerge" to get this change in? Looks like all tests passed.

PeterCaldwell commented:
Ok. Original tests failed for unclear reasons. Passed just because Luca set the "Integrate without Testing" flag. I figured that would imply Automerge so I deleted it... apparently we need both. So I just added Automerge back.

E3SM-Autotester commented:
Status Flag 'Pull Request AutoTester' - Pull Request will be Automerged

E3SM-Autotester commented:
Merge on Pull Request# 612: IS A SUCCESS - Pull Request successfully merged
